---
page: htapp-webinar-series
navText: HTAPP Webinar Series
---

<h1>Human Tumor Atlas Pilot Project (HTAPP) Webinar Series</h1>

<iframe
    width="560"
    height="315"
    src="https://www.youtube.com/embed/bEt5EYPyDHY"
    title="YouTube video player"
    frameborder="0"
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
    allowfullscreen
></iframe>

<h4>LINKS</h4>

<p>Toolbox paper</p>
<p>
    <a href="https://www.nature.com/articles/s41591-020-0844-1"
        >Slyper <i>et al</i></a
    >
    Nature Medicine volume 26, pp 792–802(2020):
    <i
        >A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen
        human tumors.</i
    >
</p>
<p>Abstract</p>
<p>
    Single-cell genomics is essential to chart tumor ecosystems. Although
    single-cell RNA-Seq (scRNA-Seq) profiles RNA from cells dissociated from
    fresh tumors, single-nucleus RNA-Seq (snRNA-Seq) is needed to profile frozen
    or hard-to-dissociate tumors. Each requires customization to different
    tissue and tumor types, posing a barrier to adoption. Here, we have
    developed a systematic toolbox for profiling fresh and frozen clinical tumor
    samples using scRNA-Seq and snRNA-Seq, respectively. We analyzed 216,490
    cells and nuclei from 40 samples across 23 specimens spanning eight tumor
    types of varying tissue and sample characteristics. We evaluated protocols
    by cell and nucleus quality, recovery rate and cellular composition.
    scRNA-Seq and snRNA-Seq from matched samples recovered the same cell types,
    but at different proportions. Our work provides guidance for studies in a
    broad range of tumors, including criteria for testing and selecting methods
    from the toolbox for other tumors, thus paving the way for charting tumor
    atlases.
</p>

<h4>Protocols</h4>

<p>
    Protocols.io links to protocols discussed in the webinar and other protocols
    included in
    <a href="https://www.nature.com/articles/s41591-020-0844-1">
        Slyper et al</a
    >
    are provided below.
</p>

<p><i>Nuclei isolation from frozen tissue for single-nucleus RNA-Seq</i></p>

<ol>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbdj2i6"
            >HTAPP_NST- Nuclei isolation from frozen tissue</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbcj2iw"
            >HTAPP_TST- Nuclei isolation from frozen tissue</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbej2je"
            >HTAPP_CST- Nuclei isolation from frozen tissue</a
        >
        V.2
    </li>
</ol>

<p><i>Fresh tissue processing protocols for single-cell RNA-Seq</i></p>

<ol>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbjj2kn"
            >HTAPP_Dissociation of human metastatic breast cancer core needle
            biopsy to a single-cell suspension for single-cell RNA-seq</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbgj2jw"
            >HTAPP_Dissociation of human primary lung cancer resection to a
            single-cell suspension for single-cell RNA-seq</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bjxjkpkn"
            >HTAPP_Depletion of CD45+ cells from single-cell suspension for
            single-cell RNA-seq</a
        >
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbhj2j6"
            >HTAPP_Dissociation of human ovarian cancer resection to a
            single-cell suspension for single-cell RNA-seq</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbkj2kw"
            >HTAPP_Processing human ovarian cancer ascites to a single-cell
            suspension for single-cell RNA-seq</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.bhbij2ke"
            >HTAPP_Depletion of CD45+ cells from ovarian cancer ascites single
            cell suspensions for single-cell RNA-Seq</a
        >
        V.2
    </li>
    <li>
        <a href="http://dx.doi.org/10.17504/protocols.io.98ah9se"
            >HTAPP_Dissociation of human neuroblastoma tumors to a single-cell
            suspension for single-cell RNA-seq using Papain</a
        >
    </li>
</ol>

<h4>
    WEBINAR 1: Tissue acquisition workflows and sc/snRNA-Seq processing
    protocols
</h4>
<p>December 3rd, 2020</p>

<h4>AGENDA</h4>
<ol>
    <li><strong>HTAPP overview</strong></li>
    <li><strong>Fresh tissue acquisition workflow</strong></li>
    <li>Key components of a tissue acquisition and allocation workflow</li>
    <li>Key aspects to keep in mind for each component</li>
    <li><strong>Fresh tissue processing for sc-RNAseq</strong></li>
    <li>Description of the processing workflow</li>
    <li>Guidelines for testing and selecting protocols</li>
    <li><strong>Frozen tissue processing for snRNA-Seq</strong></li>
    <li>Description of the processing workflow</li>
    <li>Guidelines for testing and selecting protocols</li>
    <li><strong>Single-cell vs single nucleus RNA-Seq processing</strong></li>
    <li><strong>Lessons learned &amp; challenges</strong></li>
    <li><strong>Q&amp;A</strong></li>
</ol>

<h4>PRESENTERS</h4>
<p>
    <strong>Judit Jané-Valbuena, Ph.D</strong>, Scientific Manager, Klarman Cell
    Observatory, Broad Institute
</p>
<p>
    <strong>Sébastien Vigneau, Ph.D</strong>. Head of the Innovation Lab, Center
    for Cancer Genomics, Dana-Farber Cancer Institute
</p>
<p>
    <strong>Michal Slyper, Ph.D</strong>. Research Scientist, Klarman Cell
    Observatory, Broad Institute
</p>
